AIPrompt docs (#1883)
telerik/blazor-docs: components/aiprompt/views/output.md

---
title: Output View
page_title: AIPrompt Output View

> description: Explore the AIPrompt Output View.

Reviewer: The meta descriptions should be a bit longer to bring SEO value.

> description: Explore the AIPrompt Output View. Learn how to handle output rating.

Reviewer 2: ```suggestion
description: Explore the AIPrompt Output view that shows the responses generated by the AI service, and learn how to handle output rating.
```

slug: aiprompt-views-output
tags: telerik,blazor,aiprompt,ai,prompt,output
published: True
position: 20
---

# AIPrompt Output View

> The Output view shows the generated responses by the underlying AI service. Each response renders in its dedicated output card and provides two options to the user - to copy the content of the response or to retry the request.

Reviewer 2: ```suggestion
The Output view shows the responses generated by the underlying AI service. Each response renders in its dedicated output card and provides two options to the user—to copy the content of the response or to retry the request. The Output view is activated through interaction—once the user fills out a prompt within the Prompt view and requests a response, the Output view will become active.
```

> The Output view visualizes the generated responses by the underlying AI service. Each response is displayed in its dedicated output card and features two options for the user - to either copy the content of the response or to request a retry, triggering a new prompt request.

Reviewer: ```suggestion
The Output view shows the generated responses by the underlying AI service. Each response renders in its dedicated output card and provides two options to the user - to copy the content of the response or to retry the request.
```

> If the `ShowOutputRating` is enabled on the component level, the output card will also feature two additional options - to upvote and downvote the response. This interaction can be handled through the `OnOutputRate` event. For more information on how to handle the event, refer to the [AIPrompt events]({%slug aiprompt-events%}) article.

Reviewer 2: ```suggestion
If the `ShowOutputRating` is enabled on the component level, the output card will also feature two additional options—to upvote or downvote the response. To handle this interaction, you can use the `OnOutputRate` event. For more information on how to handle the event, refer to the [AIPrompt events]({%slug aiprompt-events%}) article.
```

> If you have enabled the `ShowOutputRating` parameter on the component level, the output card will also feature two additional options - to upvote and downvote the response. This interaction could be handled through the `OnOutputRate` event. For more information in regard to how to handle the event, refer to the [AIPrompt events]({%slug aiprompt-events%}) article.

Reviewer: ```suggestion
If `ShowOutputRating` is enabled on the component level, the output card will also feature two additional options - to upvote and downvote the response. This interaction can be handled through the `OnOutputRate` event. For more information on how to handle the event, refer to the [AIPrompt events]({%slug aiprompt-events%}) article.
```
